code/site/components/com_pages/class/locator/extension.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Joomlatools Pages
+ *
+ * @copyright   Copyright (C) 2018 Johan Janssens and Timble CVBA. (http://www.timble.net)
+ * @license     GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link        https://github.com/joomlatools/joomlatools-pages for the canonical source repository
+ */
+
+class ComPagesClassLocatorExtension extends KClassLocatorAbstract
+{
+    protected static $_name = 'ext';
+
+    public function locate($classname, $basepath = null)
+    {
+        if (substr($classname, 0, 9) === 'Extension')
```

Comment:
@johanjanssens what do you think of a namespace like `Pages\Extensions\` here? Should make feature refactors easier.

Reply:
@ercanozkaya As discussed in Kodekit we won’t need an additional locater. Kodekit can handle this out of the box. Technically this would be userland code and not be namespaced in Kodekit. Leaving this for now, will look into how to best solve it when i move things into Kodekit.